docs/about/contributing/SECURITY.md

```diff
@@ -0,0 +1,12 @@
+# Security Policy
+
+Coder welcomes feedback from security researchers and the general public to help improve our security.
+If you believe you have discovered a vulnerability, privacy issue, exposed data, or other security issues
+in any of our assets, we want to hear from you.
+This policy outlines steps for reporting vulnerabilities to us, what we expect, what you can expect from us.
```

Comment:
I think we should drop this line, because we don't actually talk about what to expect.

This is covered in the security policy that we link to and this line is repeated there.